numpy/lib/polynomial.py

```diff
@@ -631,7 +631,7 @@ def polyval(p, x):
        to zero) from highest degree to the constant term, or an
        instance of poly1d.
     x : array_like or poly1d object
-       A number, a 1D array of numbers, or an instance of poly1d, "at"
+       A number, an array of numbers, or an instance of poly1d, "at"
```

Comment:
Yeah, I was wondering about that too.  I didn't touch it though since I was just focusing on fixing the array part of the documentation.  I think ```at``` is fine though, just without quotes.